pkg/systemd/services.html

```diff
@@ -92,42 +92,30 @@
       <div class="panel-heading">{{Unit.Description}}</div>
       <div class="list-group">
         <div class="list-group-item">
-          <table>
-            <tr>
-              <td>
-                <span>{{Unit.ActiveState}}</span>
-                ({{Unit.SubState}})
-                <br/>
-                {{Since}}
-              </td>
-              <td>
-                {{{UnitButton}}}
-              </td>
-            </tr>
+          {{#HasLoadError}}
+            <div class="alert alert-danger">
+              <span class="pficon pficon-error-circle-o"></span>
+              <strong>{{Unit.LoadState}}</strong> Reason: {{LoadError}}
```

Comment:
"Reason" needs to be translated, or ideally it could be skipped entirely. This is already broken in master though, so ok to do it as a follow-up.

Reply:
Fixed by removing the string "Reason", since it didn't add any value.